CONTRIBUTING.md

```diff
@@ -7,6 +7,8 @@ To report a bug:
 
 Submit a `1.2.3` version upgrade for the `foo` formula:
 
+- First check if someone else has already submitted the same upgrade, since it can take time to be tested and incorporated. Look at the open [Pull Requests](https://github.com/Homebrew/homebrew/pulls) and add the formula name to the filter box at the top of the list, so it looks like `is:pr is:open foo`. If something shows up, check to see if it accomplishes what you were about to work on. If nothing matched, then proceed.
```

Comment:
All right, I took a crack at making it much more concise. My only concern is that the “foo“ pattern matches a bunch of PRs, and I am not sure people will know to look at the filter box and change the search to match their actual formula. But I guess we can give people the benefit of the doubt?